source/server/options_impl.cc

```diff
@@ -92,6 +108,9 @@ OptionsImpl::OptionsImpl(std::vector<std::string> args,
   TCLAP::SwitchArg log_format_escaped("", "log-format-escaped",
                                       "Escape c-style escape sequences in the application logs",
                                       cmd, false);
+  TCLAP::ValueArg<bool> log_format_prefix_with_location(
```

Comment:
I *think* that `TCLAP::ValueArg<bool>` should be `TCLAP::SwitchArg`.

Reply:
I didn't find a way to disable `TCLAP::SwitchArg` if it defaults to `true`. Please note that default will be flipped in one envoy release or so.

Reply:
Oh, I see. Fine to leave as-is then.